Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1
+FROM ruby:3.1.3
+WORKDIR /crimethinc
+COPY . .
+RUN apt-get update -qq && apt-get install -y nodejs postgresql-client ruby-full git curl libssl-dev libreadline-dev zlib1g-dev autoconf bison build-essential libyaml-dev libreadline-dev libncurses5-dev libffi-dev libgdbm-dev
```

Comment:
You probably don't need to install `ruby-full` since the base image is `ruby:3.1.3`. In fact, I'd double-check if you don't only need `nodejs` and `postgresql-client`.

Reply:
Oh hey, good catch! I'm really not sure how I ended up with all of those dependencies. Must have been while I was tinkering around trying to get this to work. You were right, we only need `nodejs` and `postgresql-client` apparently.